base/set.jl

```diff
@@ -282,29 +284,42 @@ julia> issubset([1, 2, 3], [1, 2])
 false
 ```
 """
-function issubset(l, r)
+function ⊆(l, r)
     for elt in l
         if !in(elt, r)
             return false
         end
     end
     return true
 end
-
 # use the implementation below when it becoms as efficient
 # issubset(l, r) = all(_in(r), l)
 
-const ⊆ = issubset
-⊊(l::Set, r::Set) = <(l, r)
-⊈(l::Set, r::Set) = !⊆(l, r)
-⊇(l, r) = issubset(r, l)
-⊉(l::Set, r::Set) = !⊇(l, r)
-⊋(l::Set, r::Set) = <(r, l)
+const issubset = ⊆
+
+"""
+    issetequal(a, b)
+
+Determine whether `a` and `b` have the same elements. Equivalent
+to `a ⊆ b && b ⊆ a`.
+
+# Examples
+```jldoctest
+julia> issetequal([1, 2], [1, 2, 3])
+false
+
+julia> issetequal([1, 2], [2, 1])
+true
+```
+"""
+issetequal(l, r) = length(l) == length(r) && l ⊆ r
```

Comment:
I think this should be the implementation for `==(::AbstractSet, ::AbstractSet)`; then we don't need `issetequal`.